### PR TITLE
AUTOPAY-28786 Update http-signatures library to use RFCs instead of Drafts

### PR DESCRIPTION
**Pull request by tikre2910**
Updated JavaDocs

### CODE REVIEW

File: legal/ADDITIONAL_LICENSE_INFO [hunk 1]
```diff
@@ -1 +1 @@
-../java.base/ADDITIONAL_LICENSE_INFO
\ No newline at end of file
+Please see ..\java.base\ADDITIONAL_LICENSE_INFO
```
Comment (tikre2910) [inline legal/ADDITIONAL_LICENSE_INFO:1]: This was added/ changed by IntelliJ's JavaDoc tool.. Keep or undo?

File: legal/ASSEMBLY_EXCEPTION [hunk 2]
```diff
@@ -1 +1 @@
-../java.base/ASSEMBLY_EXCEPTION
\ No newline at end of file
+Please see ..\java.base\ASSEMBLY_EXCEPTION
```
Comment (tikre2910) [inline legal/ASSEMBLY_EXCEPTION:1]: This was added/ changed by IntelliJ's JavaDoc tool.. Keep or undo?

File: legal/LICENSE [hunk 3]
```diff
@@ -1 +1 @@
-../java.base/LICENSE
\ No newline at end of file
+Please see ..\java.base\LICENSE
```
Comment (tikre2910) [inline legal/LICENSE:1]: This was added/ changed by IntelliJ's JavaDoc tool.. Keep or undo?

File: legal/ADDITIONAL_LICENSE_INFO (hunk 1 above)
Comment (robelcik) [reply legal/ADDITIONAL_LICENSE_INFO:1]: It looks like it depends on OS (Linux vs Windows) and JDK version/flavour. I'd say do as you wish, it doesn't really matter. Same for similar changes.

**Review by robelcik — APPROVED** [review]

**Pull request merged**